pkg/webhook/patch.go

```diff
@@ -401,6 +401,7 @@ func addContainerPort(pod *corev1.Pod, port int32, protocol string) *patchOperat
 
 	path := fmt.Sprintf("/spec/containers/%d/ports", i)
 	containerPort := corev1.ContainerPort{
+		Name:          "metrics",
```

Comment:
The function `addContainerPort` is a generic one and not specific to the Prometheus port. What you probably should do is pass in the port name and specify the port name in the caller.

Reply:
@liyinan926 
Hello,
There is a suggestion for this. https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/issues/1101
After your review is made, I will fix it.
Thanks!

Reply:
@liyinan926 Added port name from caller